modules/org/infinispan/dependencies/jdk/openjdk/trim.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-set -e
+#set -e
 DIR=$(dirname $0)
 PACKAGES=$(cat $DIR/packages.txt)
-rpm -e --nodeps $PACKAGES
+for pkg in $PACKAGES
+do
+    cmd=`rpm --verify $pkg 2> /dev/null`
+    rc=$?
+    if [[ $rc -eq 0 ]]
```

Comment:
There's no need for the `cmd` and `rc` values here, as bash allows a command to be executed directly in the if statement. For example:

```
if rpm --verify $pkg 2> /dev/null; then
  ...
fi
```